tests/www/apply/test_detail_rdv_insertion.py

```diff
@@ -67,68 +78,118 @@ def setup_method(self, freeze, **kwargs):
             for_snapshot=True,
         )
 
-    def test_details_should_not_include_appointments_tab_is_not_configured(self, client):
+    @pytest.mark.parametrize(
+        "profile,view_name,template_name",
+        [
+            ("employer", "details_for_company", "process_details_company"),
+            ("job_seeker", "details_for_jobseeker", "process_details"),
+        ],
+    )
+    def test_details_should_not_include_appointments_tab_if_not_configured_and_without_upcoming_appointments(
+        self, client, profile, view_name, template_name
+    ):
         self.job_application.to_company.rdv_solidarites_id = None
         self.job_application.to_company.save()
         self.participation.appointment.delete()
 
-        client.force_login(self.job_application.to_company.members.get())
-        response = client.get(
-            reverse("apply:details_for_company", kwargs={"job_application_id": self.job_application.pk})
-        )
-        assertTemplateUsed(response, "apply/process_details_company.html")
+        self._force_profile_login(client, profile)
+        response = client.get(reverse(f"apply:{view_name}", kwargs={"job_application_id": self.job_application.pk}))
+        assertTemplateUsed(response, f"apply/{template_name}.html")
         assertNotContains(response, self.APPOINTMENTS_TAB_TITLE)
 
-    def test_details_should_include_appointments_tab_is_not_configured_and_has_upcoming_appointments(self, client):
+    @pytest.mark.parametrize(
+        "profile,view_name,template_name",
+        [
+            ("employer", "details_for_company", "process_details_company"),
+            ("job_seeker", "details_for_jobseeker", "process_details"),
```

Comment:
Petit détail, mais j’aurais tendance à laisser le nom complet pour faciliter le grep au lieu d’interpoler la moitié.
```suggestion
            ("employer", "apply:details_for_company", "process_details_company"),
            ("job_seeker", "apply:details_for_jobseeker", "process_details"),
```